### PR TITLE
Add example of larger floating farm with comparison to fixed-bottom.

### PR DESCRIPTION
**Pull request by misi9170**
Ready for review.

Address #670 (request for more realistic example of floating wind farm, including turbine powers and visualization).

Note: uses "default" EmGauss parameters, which are subject to change and should be tuned to match observations from a particular farm.

TODO:
- [X] Create example, including necessary input files
- [X] Check tests, ruff, isort pass
- [x] Update examples index in documentation

### CODE REVIEW

**Review by rafmudaf — APPROVED** [review]

**Comment by rafmudaf** [comment]
Maybe one thing to add is how examples 24 and 29 are different. As a reader, I can guess that 29 has a larger problem size, but otherwise why would I need to go through example 24 if I can study 29?

**Comment by misi9170** [comment]
> Maybe one thing to add is how examples 24 and 29 are different. As a reader, I can guess that 29 has a larger problem size, but otherwise why would I need to go through example 24 if I can study 29?

Right, we now have three examples for floating, each looking at something slightly different:
- 24 looks at how tilt affects turbine power and thrust curves, but does not include any wake effects
- 25 looks at how tilt affects wake deflections, but does so by essentially hard-coding the tilt angle (so is useful for understanding, but is not very realistic)
- 29 is a more realistic situation where the tilt is a function of wind speed. Both Cp/Ct curve effects and wake effects are included. So, it's the best example of how we would handle a real floating farm.

**Comment by rafmudaf** [comment]
> @rafmudaf I've now provided a small update to the Examples Index to reflect this.

Excellent, thanks!

File: docs/examples.md [hunk 1]
```diff
@@ -97,7 +97,11 @@ of a turbine layout within FLORIS.
 
 ### 24_floating_turbine_models.py
 Demonstrates the definition of a floating turbine and how to enable the effects of tilt
-on Cp and Ct.
+on Cp and Ct.
```
Comment (rafmudaf) [inline docs/examples.md:100]: You've got a trailing space here

**Review by bayc — APPROVED** [review]
Looks great @misi9170 ! I know you are looking into the divide by zero warning on the EmG model, but otherwise good to go from my side!

**Pull request merged**